plasma_framework/package.json

```diff
@@ -6,8 +6,10 @@
     "test": "test"
   },
   "dependencies": {
-    "openzeppelin-solidity": "^2.3.0",
-    "dotenv": "^8.0.0"
+    "@truffle/hdwallet-provider": "^1.0.19",
+    "dotenv": "^8.0.0",
+    "lodash.clonedeep": "^4.5.0",
+    "openzeppelin-solidity": "^2.3.0"
```

Comment:
Let's lock down the openzeppelin-solidity version and upgrade manually.

Reply:
Done in [this commit](https://github.com/omisego/plasma-contracts/pull/298/commits/57b5c0ab53f612c24d13118bbc75448a20eeff29).